### ISSUE TITLE
Set up user login for the website

### ISSUE THREAD

**Issue by hao-fang** [acl-org/acl-2020-virtual-conference#13]
We will require auth to access the website. This might depend on #5, but to start with, we can just set up everything locally with a mock user account.

- [x] create a login front-end interface
- [x] enable auth to access the website

Ideally, we may want a common login cross different tools, but this may be out of scope, unless there is an easy solution.

Related Issue: https://github.com/Mini-Conf/Mini-Conf/issues/32

**Comment by hao-fang** [comment]
Re common login, it seems that RocketChat supports some OAuth which we may use.
https://docs.rocket.chat/administrator-guides/authentication/oauth

For now I think we won't need login for SlidesLive and Zoom, but we will have to confirm.

**Comment by hao-fang** [comment]
Re login with static website -- It seems AWS has a nice solution [here](https://douglasduhaime.com/posts/s3-lambda-auth.html) with S3 and Lambda.
And S3 and Lambda are both inexpensive.
Not sure if DO has something similar we can use.

**Comment by hao-fang** [comment]
UPDATE: It seems this is the one-stop solution: https://aws.amazon.com/blogs/networking-and-content-delivery/authorizationedge-how-to-use-lambdaedge-and-json-web-tokens-to-enhance-web-application-security/

**Comment by hao-fang** [comment]
UPDATE: Should use this one which uses signed cookies instead of signed URLs
https://aws.amazon.com/blogs/networking-and-content-delivery/authorizationedge-using-cookies-protect-your-amazon-cloudfront-content-from-being-downloaded-by-unauthenticated-users/

**Comment by hao-fang** [comment]
OAuth RocketChat with AWS Cognito: https://github.com/RocketChat/Rocket.Chat/issues/14866#issuecomment-513340416
